_content/excerpts/pali-literature-in-cambodia_saddhatisa-hammalawa.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Pali Literature in Cambodia"
+authors:
+  - "Hammalawa Saddhatisa"
+source_url: "https://archive.org/details/saddhatissa2004paliliteratureofsoutheastasia/mode/2up"
+drive_links:
+  - "https://drive.google.com/file/d/1js6qs5UpABickIjKukS3HWxrl8oytuNB/view?usp=sharing"
+course: cambodian
+tags:
+  - pali-literature
+year: 2004
```

Comment:
This was originally published as an article in the JPTS vol 9 in 1981.

Here's the external_url: "https://archive.org/download/jpts-ix-1981/Pali%20Literature%20in%20Cambodia%20-%20H%20Saddhatissa_text.pdf"

Please update the rest of the metadata accordingly.